homeassistant/components/deconz/__init__.py

```diff
@@ -121,10 +127,27 @@ def async_configure(call):
         See Dresden Elektroniks REST API documentation for details:
         http://dresden-elektronik.github.io/deconz-rest-doc/rest/
         """
-        deconz = hass.data[DOMAIN]
         field = call.data.get(SERVICE_FIELD)
+        entity_id = call.data.get(SERVICE_ENTITY)
         data = call.data.get(SERVICE_DATA)
-        yield from deconz.async_put_state(field, data)
+        deconz = hass.data[DOMAIN]
+
+        registry = hass.data.get(DATA_REGISTRY)
+        if registry.async_is_registered(entity_id):
+            entity = registry.async_get_entry(entity_id)
+            groups = list(deconz.groups.values())
+            lights = list(deconz.lights.values())
+            sensors = list(deconz.sensors.values())
+            devices = groups + lights + sensors
```

Comment:
I think what you're looking for is `chain`, that way you don't have to create a list but can just iterate over it.

```python
from itertools import chain

for device in chain(deconz.groups.values(), deconz.lights.values(), …):
```